the_big_bang.ipynb

```diff
@@ -0,0 +1,295 @@
pY7+z6mGHmrhQPlEvz3y/77hpKXSnAllirwg6T2QbMSpuyAG5YkWTRvdASb2lPsocopTK+MKxaLCP7ZpBwjIFTABjyg07Ect8d1NTReZTmhQjWwDdRHxmlQxZY4pnFkMjOAoianQoTni/UGDt2jhYcfG743PeMDFxy0IgwVBWCCjRvXoJuixZ+pjI8b+Tbnkgnjw4w/AD+R/nxw5KJHj8yfE33xi8emyj11ndfwzHsJbZZsNLoL5kzXHD5S2/SLD3MaJcFEUIdC6Tj5j0oAIQ3A8OXun9/GAF0NQMB63u/VxAmdAaBP8R87c7CixhiuzV4JqHuPQpY3TtI4Gznp7vG6UAgU2BfLzPkY8Wq9lIJ6P8ANNxvl7R3bk557C+sFoSJNXRfxfncmG7AFQa0gPK1/SQyzx78Od+Atej1lzaTv2TA3ST6tmesOIqXgJb8mu3CNRaBdXl8X48ZUWBdvNawf8nDxrNwmu+/j6Mru6s/wM3ezesH/Zf8ecAT6P7YVwfN/vklBZ9AX8BS/vWVXLD2h0KIWMUAgsuF5dPDdcmAkiASbY8e7P7zs34a/fj/AEFo1wxdfx+McSOnv/rG+EMG6i4CocBNCQeaeL6zQSaCEegOFvOh84qValNhDeBf2BlHTUz8Cej8r9sZl3hNdT2rDfv0bEMFajUKnlul8xr2fkwM8HGvdw0msRE18z6PrmbdBEpRwql1/nMK+XBwd3EECDaB/wCg1Qkxe7hLTtGhXNCNJgXNNk4LRtA7hyZpTpMHHw9vsoK1IRUyDkQrQmTem/NuG03UsfqswyFOXmEcdYtSP2oYfIet5gxZZ62th034oeTZTepPioL8E9bJvbckBNYtOUx+fzN9y8PKj8oCge3NwimdGxSH5umHPe9YwPw/BeVODTe+Wwx42a00Ouv03147Yr+8mHgN2+fWiGRpJyLF1EWffr240tC6B42EN2CNQ3k1UZroKNPzCawZhElCVTTA8QIoOMPSXt0JtTRtj3nZ5v5jCvZ7xoHgXuTaXxD4/rnQCABHjwKfR5G4JUCGhsgXjf8ALj5AFWdhwdWbvyy8SWU2E4aY+V58uGkP1E8hbX2gnjYaJoIT8tE7862YAZordWylojru/wAZvqzmhIxOf8jbrIW8bCPwQ++/F15NDhsJET5Od+DXJoVh0teDxDT4MN4VaIwDnHQeP65vWo3gjkH8H264JEvdACbZ/wBv5yIDK1G3nrydrCYq8ejw+EZv/rJcV9eMC034yA+gZfyZqDuRO/DgDdh6X9POb0k8pD9ZBRZDwdn+S49igfPx/TOj7xz89xcffnFgxHxP+s1Z0Ar6T3+cmzZEaUKBe+BqY6xJT7SjlhpTSjUuF5sLaVlZL4BUJkZaNyHJXdugBPW4gPtSNhz8Hg84uAqqxnh29kPnWEbxxX3r39v7YwSki99x9r/fKdAAhlM2PKT53thmwDR9Kmhgo9G3/oJiZomw2JRzS0ot8QWugFBakO/YElYigOqMxeVVXXmTDSgIcFJ3DJ9LYai+JLcNgyQv2T6h0im64FUKnSjPTfyj30K7PKrpzvkIAY0lQgZOwQwWGdG90Hr24+jeRrw/u9e9jl3AT9Ihz/Ec+VrmTM7AIbUogUI2D7Hw+Y8efvNH0gMl35iWu+TNKyGyaGkem33zBkbhTfkk6d/LxlKFtq0epVOgY5vPbqwRtrPSENF7nVx1KoP9pvZWRZwxNRvruzR/ZwiE3ASdmukSr7esHzA7cq2xNk/XnJplUVyQmH2OL5bkRCEIqsv8/viZxwGGw2Wtjm/ZeZyH2nVHgloGvU+nJ2SQREf3G3+bzNUtdjuwENJD5k/CHOACnA+cT16DWd/xV7+TcPmyC5M1Q5U4wpNHv1M1Udw1Pu3T/HGZEgg8bT4Gvv7z0QiFE2Pp9YTqdQIPmfs7UyIKh1HGHjyHwwygpKnmgyVeh1mK8y0dXdVnC3+DPMdlgGfVfWKHbd8a+mbMKbp9l1++YnolOFHi+3FV+B7v5/y4p44nj78/ExZ9htBUuDev3OOHnjYN6IU/IrNcnuqvzzzycHAGf6HmVOX+v69Ybersd9/WHRNCwootq/2xURIgvJhBiYmVrYB8f0yCqiCmo8wp/DYaGvW6E4b3k0VERS+6P4YYDlBX7Zv/ACeS8j7VE7H+aDOXiadtD38tfvJXhKEHtDwD85QI7pMOG5XzQr/6D6J5LfPgk8pdgGMjjyvC4MTC2sMFTVPSBSkx69Bx4IbcmeWI0uasjO1tA6Gp3HoExAF6hxlLzHELwi+aCWOYEGRw0cwJY5qh3yIwWsxgEVEJ1jG7MCrlGgAH0+c6gnPrQkzz/wA6Dr0pB/Aif84NcojE/CmASMm8W8QEDQNRrD4Sq91CdlI+nclR7lzo0gpNaNRfQeq5MAaDi9kTT77bvuA2Lo6EFIBDzsG+ZSZubS6FhrR6yi1EtIdpeqnFLZYXQgoiQh/jg3gAQ0IWjxdrrQ5pzLGAQ1fdBfNDLA0COcAGtiPyvDA1ANkAsJPIJ988YtiwUiAETiHzmiPjBUAz+CZ0G/D3nrGFEio4Lq8P2e8Jt3ZPDVD9q+8RxhKLeoIXR38py5p1DFBVNLL8dM7gh0+Eodw0P2XLKX4KQV2P+Q3j0/EOKeL7cD6vdK+a/wAI03gVia0n5olf7XG37BQ49ATS6+jmFeoewtFiqZ2cJ5M5CKKdnvI51oH6a4DsAYEH3UMRPCLEb9T4z8rJyYSb8vcGz0/6rSzJKAik+UT5Nf39YL7Wn3/tnGZhtH1rIqQHuAc3OaFAoaa4W1nnPD/R+MfhudGKbvgVeLCLREdXFhlqoqgp0g7CxmS+UhVSn4AHy0ZHMC8U8X2/SmT65NyJTY3v2u8XhvlStbPjn58zAaV9nvxIUv1DIA7HweXXTvH6w15B2Epur7n34c7XU5SRB9qEC0+yGaxd5SOqVf8AfMXh21okj5NQSC6+ZiPdTqvCjiNZlVWp/wA87DlG+rTsQ63tk3Osh/
```

Comment:
It's kind of tricky, because the solution now implies that you have to use one object, and I expected some candidates to fail to solve it with one object... WDYT?